app/config/config.js

```diff
@@ -13,6 +13,8 @@ export const PAGE_SIZE = {
 
 export const LIVE_SEARCH_DEBOUNCE_TIME = 300;
 
+export const PUBLICATION_REPORT_START_YEAR = 1981;
```

Comment:
Dit is eigen invulling: ik had opgezocht dat de eerste Vlaamse Regering 22/12/1981 van start ging. Het was maar om totaal onmogelijke invoer te vermijden maar niet te restrictief te zijn. Mocht men hier toch een globale constant voor willen maken, misschien eerder iets noemen als `START_FIRST_FLEMISH_GOVERNMENT`.

Reply:
Ik wist zelf niet goed wat ik daar moest nemen van naam. Voor mij is het gelijk wat we daar nemen